transport/tchannel/handler.go

```diff
@@ -137,6 +141,23 @@ func (h handler) callHandler(ctx context.Context, call inboundCall) error {
 	}
 	treq.Headers = headers
 
+	// Extract opentracing etc baggage from headers
+	var spanCtx opentracing.SpanContext
+	carrier := opentracing.TextMapCarrier(treq.Headers.Items())
+	spanCtx, _ = h.Tracer.Extract(opentracing.TextMap, carrier)
+
+	span := h.Tracer.StartSpan(
```

Comment:
I wonder if you simply want to bypass doing anything tracing for tchannel since it will do it internally anyway now.